src/bfx_tools/samtools.ml

```diff
@@ -150,6 +154,7 @@ let sort_bam_no_check ~(run_with:Machine.t) ?(processors=1) ~by input_bam =
     | `Read_name -> "sort" :: "-n" :: command
   in
   do_on_bam ~run_with input_bam ~product ~make_command
+    ~more_requirements:[`Memory `Big]
```

Comment:
@arahuja `` `Big`` (or any requirement) is instanciated by the user when they setup their `Machine.t`.